### PR TITLE
Bugfix extension build scripts

### PR DESCRIPTION
**Pull request by livecodeali**
This patch will get extension packages building on Windows again

### CODE REVIEW

**Comment by livecodeali** [comment]
I think since all these functions are used during build / running IDE, they will be well-tested for now.

**Comment by livecodeali** [comment]
Check branch: https://github.com/livecode/livecode/pull/6365

File: extensions/script-libraries/extension-utils/extension-utils.livecodescript [hunk 1]
```diff
@@ -0,0 +1,1634 @@
...
+*/
+
+/**
+Title: Extension Package Handling Utilities
+
+Type: library
+
+Summary: This library provides handlers for managing livecode extension packages
+
+Version: 1.0.0
```
Comment (montegoulding) [inline extensions/script-libraries/extension-utils/extension-utils.livecodescript:27]: @livecodeali do we have a flag we can add here to indicate this is an IDE library?

File: extensions/script-libraries/extension-utils/extension-utils.livecodescript [hunk 2]
```diff
@@ -0,0 +1,1634 @@
...
+      if the result is not empty then
+         throw "Circularity in dependencies starting from" && tToVisit
+         return empty
+      end if
+   end repeat
+   return tOrder
+end __dependencyOrder
+
+private function __extensionIsBuiltin pID
+   switch pID
```
Comment (montegoulding) [inline extensions/script-libraries/extension-utils/extension-utils.livecodescript:267]: @livecodeali this seems like something we might want to add an enhancement request for a function

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 2a37058

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 2a37058243a813a4b7c9df92ead36951856098d0

**Comment by livecode-vulcan** [comment]
😎 test success 2a37058243a813a4b7c9df92ead36951856098d0

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1261)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1277)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1251)
* try-community-universal-ios-iphoneos11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.2/builds/204)
* try-community-universal-ios-iphonesimulator11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.2/builds/202)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1255)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1281)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1261)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1276)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1260)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1270)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1312)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1251)

File: extensions/script-libraries/extension-utils/extension-utils.livecodescript (hunk 1 above)
Comment (livecodeali) [reply extensions/script-libraries/extension-utils/extension-utils.livecodescript:27]: Not yet I don't think...

**Pull request merged**